doofinder-for-woocommerce/assets/js/wizard.js

```diff
@@ -1,117 +1,115 @@
 jQuery(() => {
-    const $ = jQuery.noConflict();
+  const $ = jQuery.noConflict();
 
-    let connectModal;
+  let connectModal;
 
-    const popupCenter = ({ url, title, w, h }) => {
-        const dualScreenLeft =
-            window.screenLeft !== undefined
-                ? window.screenLeft
-                : window.screenX;
-        const dualScreenTop =
-            window.screenTop !== undefined ? window.screenTop : window.screenY;
-        const width = window.innerWidth
-            ? window.innerWidth
-            : document.documentElement.clientWidth
-            ? document.documentElement.clientWidth
-            : screen.width;
-        const height = window.innerHeight
-            ? window.innerHeight
-            : document.documentElement.clientHeight
-            ? document.documentElement.clientHeight
-            : screen.height;
-        const systemZoom = width / window.screen.availWidth;
-        const left = (width - w) / 2 / systemZoom + dualScreenLeft;
-        const top = (height - h) / 2 / systemZoom + dualScreenTop;
-        const newWindow = window.open(
-            url,
-            title,
-            `
+  const popupCenter = ({ url, title, w, h }) => {
+    const dualScreenLeft =
+      window.screenLeft !== undefined ? window.screenLeft : window.screenX;
+    const dualScreenTop =
+      window.screenTop !== undefined ? window.screenTop : window.screenY;
+    const width = window.innerWidth
+      ? window.innerWidth
+      : document.documentElement.clientWidth
+      ? document.documentElement.clientWidth
+      : screen.width;
+    const height = window.innerHeight
+      ? window.innerHeight
+      : document.documentElement.clientHeight
+      ? document.documentElement.clientHeight
+      : screen.height;
+    const systemZoom = width / window.screen.availWidth;
+    const left = (width - w) / 2 / systemZoom + dualScreenLeft;
+    const top = (height - h) / 2 / systemZoom + dualScreenTop;
+    const newWindow = window.open(
+      url,
+      title,
+      `
         scrollbars=yes,
         width=${w / systemZoom},
         height=${h / systemZoom},
         top=${top},
         left=${left}
       `
-        );
+    );
 
-        if (window.focus) {
-            newWindow.focus();
-        }
+    if (window.focus) {
+      newWindow.focus();
```

Comment:
This shorthand is deprecated since jQuery 3.3. https://api.jquery.com/focus-shorthand/. A better option could be:

```suggestion
      newWindow.trigger("focus");
```

Reply:
I changed this but this is not using a jquery object, it is calling the [window.focus()](https://developer.mozilla.org/en-US/docs/Web/API/Window/focus) function to bring the new window to the front.

Reply:
Oh, you are right! I was wrong here, forget my comment then :pray: